core/server/ServerCore.js

```diff
@@ -3,8 +3,9 @@ const collisions = require("../utils/collisions");
 const CharFactory = require("../state/CharFactory");
 const LoopActionsQ = require("./LoopActionsQ");
 const DemoWish = require("./wishes/DemoWish");
+const WS_ACTIONS = require("../WS_ACTIONS");
 
-const VERSION = "0.0.5";
+const VERSION = "0.0.6";
```

Comment:
```suggestion
const VERSION = "0.0.7";
```

Reply:
Since those packages are separate - I thought it's required to have separate versions

Reply:
I propose to keep an eye on minor version to the same minors. Builds can differ IMHO)

Reply:
@Vitali8